android/libraries/rib-coroutines/src/main/kotlin/com/uber/rib/core/RibDispatchers.kt

```diff
@@ -0,0 +1,43 @@
+package com.uber.rib.core
+
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.Dispatchers
+
+object RibDispatchers {
```

Comment:
Looking for feedback on this delegate API.

Will be used to set Dispatchers at app startup like RxJavaPlugins.